src/NServiceBus.Testing.Fakes/NServiceBus.Testing.Fakes.csproj

```diff
@@ -79,6 +85,9 @@
     <None Include="..\..\packaging\nuget\nservicebus.testing.fakes.nuspec">
       <Link>nservicebus.testing.fakes.nuspec</Link>
     </None>
+    <None Include="..\Test.snk">
+      <Link>Test.snk</Link>
```

Comment:
We do, because it's a project reference. Core.Tests are strong named, and it won't compile otherwise.